setup.py

```diff
@@ -106,6 +107,7 @@ def read(fname):
                 'compile_translations': compile_translations },
       entry_points={'console_scripts': [
           'webant=webant.webant:main',
+          'webant-manage=webant.manage:main',
```

Comment:
I had the same doubt, so here's why:
* it heavily depends on webant
* we could want to include in it some web-specific subcommands

Despite this reasons, I am not very sure about the name.